Zend/zend_execute.h

```diff
@@ -44,6 +44,7 @@ ZEND_API zend_class_entry *zend_lookup_class(zend_string *name);
 ZEND_API zend_class_entry *zend_lookup_class_ex(zend_string *name, zend_string *lcname, uint32_t flags);
 ZEND_API zend_class_entry *zend_get_called_scope(zend_execute_data *ex);
 ZEND_API zend_object *zend_get_this_object(zend_execute_data *ex);
+ZEND_API zend_never_inline ZEND_COLD void ZEND_FASTCALL zend_deprecated_function(const zend_function *fbc);
```

Comment:
```suggestion
ZEND_API ZEND_COLD void ZEND_FASTCALL zend_deprecated_function(const zend_function *fbc);
```

I'd also move this somewhere else, maybe next to zend_missing_arg_error.

Reply:
Feel free to already land the deprecated method change separately.

Reply:
I'll do an interactive rebase first after you had a chance to review the changes, and then do the merge.

Reply:
The zend_never_inline here is still there :)

Reply:
Oh no, I didn't notice this suggestion, only the comment below. :) I'll fix this before committing to master